### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by alexbara2000**
Added unit tests that render all of the frontend components.
Refactored the file naming convention for frontend

### CODE REVIEW

File: client/src/__tests__/Signup.test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { render } from '@testing-library/react';
+import Signup from '../pages/Signup';
+
+test('Renders the Signup component', () => {
```
Comment (malhimaira) [inline client/src/__tests__/Signup.test.js:4]: tests look lit!

**Comment by malhimaira** [comment]
Code looks great @alexbara2000 , I'm gonna trust your deleted all the correct files, just check with Eric @leoenix to make sure nothing will be a problem :)

**Review by JeffreyCHChan — APPROVED** [review]
Looks like lots of renaming and removing of unnecessary files

**Review by leoenix — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ node_modules
 client/node_modules
 client/src/.DS_Store
 .DS_Store
+client/coverage
```
[the remark this answers is not in the document]
Comment (alexbara2000) [reply .gitignore:8]: yes that is the coverage report

**Pull request merged**